src/TimingData.cpp

```diff
@@ -18,40 +18,75 @@ static const int INVALID_INDEX = -1;
 
 TimingSegment* GetSegmentAtRow( int iNoteRow, TimingSegmentType tst );
 
-TimingData::TimingData(float fOffset) : m_fBeat0OffsetInSeconds(fOffset)
-{
-}
-
-void TimingData::Copy( const TimingData& cpy )
-{
-	/* de-allocate any old pointers we had */
-	Clear();
+void TimingData::Copy(const TimingData &other) {
```

Comment:
We don't actually need the TimingData changes here, if it's being taken care of within NotesLoader. 

We should factor in the song offset during AdjustSync's autosync feature, however!

Reply:
I see the idea to load song, optionally adjusted and use TimingData with null offset inside. I will think about this.